src/style.css

```diff
@@ -0,0 +1,121 @@
+:root {
+  --main-accent-color: #060b35;
+  --secondary-color: #616070;
+  --color-of-button: #00acdc;
+}
+
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+}
+
+.product-card {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-width: 200px;
+  display: flex;
+  flex-direction: column;
+}
+
+.product-card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.product-card__title {
+  color: var(--main-accent-color);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 0 0 4px;
+}
+
+.product-card__code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--secondary-color);
+  margin: 0 0 16px;
+}
+
+.product-card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url("images/star.svg");
+  background-size: 13px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url("images/star-active.svg");
+}
+
+.product-card__reviews {
+  color: var(--main-accent-color);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  vertical-align: middle;
+  margin: 0;
+  margin-top: 2px;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.product-card__price-title {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: var(--secondary-color);
+}
+
+.product-card__price-count {
+  color: var(--main-accent-color);
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.product-card__button {
+  background-color: var(--color-of-button);
+  padding: 11px 65px;
+  border: 1px solid var(--color-of-button);
+  border-radius: 5px;
+  text-transform: uppercase;
+  text-align: center;
+  text-decoration: none;
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  transition: all 0.2s;
```

Comment:
Don't use "all". It's not good for performance. Better specify the exact properties
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x